docs/examples/tree-v2/custom-node.vue

```diff
@@ -36,18 +40,13 @@ const createData = (
     }
   })
 }
-export default defineComponent({
-  setup() {
-    return {
-      data: createData(4, 30, 40),
-      props: ref({
-        value: 'id',
-        label: 'label',
-        children: 'children',
-      }),
-    }
-  },
-})
+
+const props = {
```

Comment:
Let's use `treeProps` to avoid ambiguity. Because we can use `const props = defineProps()` in composition api.